### PR TITLE
Header31

### PR DESCRIPTION
**Pull request by BurcuAcan**
…gn with standardization

### CODE REVIEW

File: editor-components/header/header31/header31.module.scss [hunk 1]
```diff
@@ -37,68 +37,69 @@
...
-            max-width: var(--composer-content-width);
             position: relative;
             height: 100vh;
             width: 100%;
             z-index: 1;
           .main-social {
               position: absolute;
               bottom: 20px;
-              left: 0px;
+              left: 50px;
```
Comment (Ayazelar) [inline editor-components/header/header31/header31.module.scss:59]: gap usage?

File: editor-components/header/header31/header31.module.scss [hunk 2]
```diff
@@ -107,11 +108,12 @@
               color: white;
               position: absolute;
               bottom: 20px;
-              right: 80px;
+              right: 50px;
```
Comment (Ayazelar) [inline editor-components/header/header31/header31.module.scss:111]: gap usage?

File: editor-components/header/header31/header31.module.scss [hunk 3]
```diff
@@ -120,8 +122,8 @@
               position: absolute;
               bottom: 50px;
               z-index: 4;
-              right: 25px;
-              font-size: 11vw;
+              right: 20px;
```
Comment (Ayazelar) [inline editor-components/header/header31/header31.module.scss:125]: gap usage?

File: editor-components/header/header31/header31.module.scss [hunk 4]
```diff
@@ -168,198 +167,185 @@
...
-            .slider-container{
-                .content{
-                    left: 44px;
+            .slider-content{
+                .slider-container{
+                    .carousel {
+                        width: 10%;
+                        .swiper-wrapper{
+                            .content{
+                                left: 44px;
```
Comment (Ayazelar) [inline editor-components/header/header31/header31.module.scss:279]: gap usage?

File: editor-components/header/header31/header31.module.scss [hunk 5]
```diff
@@ -168,198 +167,185 @@
...
+                .control-num {
+                    bottom: calc(var(--composer-gap-xl) + var(--composer-gap-md));
+                    right: 20px;
+                    width: fit-content;
+                    font-size: 10rem;
+                }
+    
+                .control-nav {
+                    bottom: var(--composer-gap-xl);
+                    right: 40px;
```
Comment (Ayazelar) [inline editor-components/header/header31/header31.module.scss:302]: gap usage?

File: editor-components/header/header31/header31.module.scss [hunk 6]
```diff
@@ -168,198 +167,185 @@
...
-                .subtitle{
-                    font-size: 13px !important;
+            .slider-content{
+                .slider-container{
+                    .carousel {
+                        display: none;
+                    }
+                    .slider-inner {
+                        width: unset;
+                        padding: 20px;
```
Comment (Ayazelar) [inline editor-components/header/header31/header31.module.scss:327]: gap usage?

File: editor-components/header/header31/header31.module.scss [hunk 7]
```diff
@@ -168,198 +167,185 @@
...
-                    max-width: 300px !important;
+                .main-social {
+                    bottom: 60px;
+                    left: 20px;
                 }
-                .description{
-                    font-size: 13px !important;
+                .control-num {
+                    bottom: 60px;
+                    right: 20px;
```
Comment (Ayazelar) [inline editor-components/header/header31/header31.module.scss:342]: gap usage?

File: editor-components/header/header31/header31.module.scss [hunk 8]
```diff
@@ -168,198 +167,185 @@
...
+                }
+    
+                .control-nav {
+                    top: 50%;
+                    right: 20px;
```
Comment (Ayazelar) [inline editor-components/header/header31/header31.module.scss:347]: gap usage?

File: editor-components/header/header31/header31.tsx [hunk 9]
```diff
@@ -849,32 +871,24 @@ class Header31 extends BaseHeader {
...
     const activeSlideObj = slider[activeIndex];
-    const subtitle = slider.length > 0 ? activeSlideObj?.subtitle : <></>;
-    const title = slider.length > 0 ? activeSlideObj?.title : <></>;
-    const description = slider.length > 0 ? activeSlideObj?.description : <></>;
     const textAnimationEnabled = this.getPropValue("textAnimation");
     const overlay = this.getPropValue("overlay");
 
-    const icons = this.castToObject<any>("icons");
+    const icons = this.castToObject<Icon[]>("icons");
+    console.log("activeSlideObj", activeSlideObj)
```
Comment (Ayazelar) [inline editor-components/header/header31/header31.tsx:888]: remove log

File: editor-components/header/header31/header31.tsx [hunk 10]
```diff
@@ -1043,19 +1055,10 @@ class Header31 extends BaseHeader {
...
-                            {this.getPropValue("circleIcon") && (
-                              <ComposerIcon
-                                name={this.getPropValue("circleIcon")}
-                                propsIcon={{
-                                  className: ``,
-                                  size: 25,
-                                }}
-                              />
-                            )}
                             <span className={this.decorateCSS("button-text")}>
```
Comment (Ayazelar) [inline editor-components/header/header31/header31.tsx:1058]: use base button

**Review by Ayazelar — CHANGES_REQUESTED** [review]
.

File: editor-components/header/header31/header31.module.scss (hunk 1 above)
Comment (BurcuAcan) [reply editor-components/header/header31/header31.module.scss:59]: static for container padding

File: editor-components/header/header31/header31.module.scss (hunk 2 above)
Comment (BurcuAcan) [reply editor-components/header/header31/header31.module.scss:111]: static for container padding

File: editor-components/header/header31/header31.module.scss (hunk 3 above)
Comment (BurcuAcan) [reply editor-components/header/header31/header31.module.scss:125]: static for container padding

File: editor-components/header/header31/header31.module.scss (hunk 4 above)
Comment (BurcuAcan) [reply editor-components/header/header31/header31.module.scss:279]: I changed with 40px and it is also static for container padding

File: editor-components/header/header31/header31.module.scss (hunk 5 above)
Comment (BurcuAcan) [reply editor-components/header/header31/header31.module.scss:302]: static for container padding

File: editor-components/header/header31/header31.module.scss (hunk 6 above)
Comment (BurcuAcan) [reply editor-components/header/header31/header31.module.scss:327]: static for container padding

File: editor-components/header/header31/header31.module.scss (hunk 7 above)
Comment (BurcuAcan) [reply editor-components/header/header31/header31.module.scss:342]: static for container padding

File: editor-components/header/header31/header31.module.scss (hunk 8 above)
Comment (BurcuAcan) [reply editor-components/header/header31/header31.module.scss:347]: static for container padding

**Review by Ayazelar — APPROVED** [review]

**Pull request merged**